x/foundation/msgs.go

```diff
@@ -25,7 +25,10 @@ func (m MsgFundTreasury) ValidateBasic() error {
 
 // GetSigners implements Msg.
 func (m MsgFundTreasury) GetSigners() []sdk.AccAddress {
-	signer, _ := sdk.AccAddressFromBech32(m.From)
+	signer, err := sdk.AccAddressFromBech32(m.From)
+	if err != nil { // should never happen as valid basic rejects invalid addresses
+		panic(err.Error())
```

Comment:
How about using `MustAccAddressFromBech32()`? Like:
```suggestion
	signer := sdk.MustAccAddressFromBech32(m.From)
```

FYI: `x/group` uses it.
https://github.com/cosmos/cosmos-sdk/blob/c2f0c0f78504cb225db01f5ae8dfea5b08b47bcb/x/group/msgs.go#L30

Reply:
Thank you, I use it.